src/Models/Magnetization_transfer/qmt_spgr.m

```diff
@@ -303,7 +303,7 @@ function plotModel(obj, x, data)
             Sim.Opt.AddNoise = 0;
             SPGR_PlotSimCurve(data.MTdata, data.MTdata, Protocol, Sim, SimCurveResults);
             title(sprintf('F=%0.2f; kf=%0.2f; R1f=%0.2f; R1r=%0.2f; T2f=%0.2f; T2r=%f; Residuals=%f', ...
-                x.F,x.R1f,x.R1r,x.T2f,x.T2r),...
+                x.F,x.kf,x.R1f,x.R1r,x.T2f,x.T2r,x.resnorm),...
```

Comment:
`plotmodel_test' does not use demo data. Can you check if it passes on your local ?

Reply:
It fails for me too.

I think it's because of this hack: 

https://github.com/neuropoly/qMRLab/blob/45018936fa514469ecf8ebc7a1ffdb03e09c1fde/src/Models/Magnetization_transfer/qmt_spgr.m#L286-L289

The plot model test isn't "smart" – it simply runs the plotModel method with no input. That block of code was written to handle this case (which is only ever used for tests, I'd imagine).

It assigns the "x" input argument to be the xnames, however `kf` isn't in there (it's calculated during the fit and added as a fitting output though).

Reply:
Since the whole purpose of this hack is just to give a mock plot example, maybe we should hack it further and simply add kf as another member of the struct `x`?

Reply:
Likely resnorm too?

Reply:
e.g. after `x=mat2struct(x,obj.xnames);`, add:

```

x.kf = x.F*x.kr;
x.resnorm = 0.0;

```

Reply:
`loadModel_Test` is crying now :/

```
failure: Error using AbstractModel/loadObj (line 51)
qmt_spgr:Unable to read file 'tmp.qMRLab.mat'. No such file or directory.
  AbstractModel.loadObj:51 (/home/agah/Desktop/qMRLab/src/Common/AbstractModel.m)
  save_load_creates_identical_original_object:41 (/home/agah/Desktop/qMRLab/Test/MoxUnitCompatible/quickMoxTests/AbstractModel_Test.m)
  test_main:17 (/home/agah/Desktop/qMRLab/Test/MoxUnitCompatible/quickMoxTests/AbstractModel_Test.m)
```

Reply:
Can't reproduce it.

On my MacOSx, I ran:

`res=moxunit_runtests('-recursive','-with_coverage','-cover','/Users/mathieuboudreau/neuropoly/github/qMRLab/src','-cover_exclude','*GUI*','-cover_json_file','coverage_quickMoxTests.json')`

in the folder: `/Users/mathieuboudreau/neuropoly/github/qMRLab/Test/MoxUnitCompatible/quickMoxTests`

and got

```

--------------------------------------------------

OK (passed=5)

res =

 logical

  1

```